chromium_src/third_party/blink/renderer/modules/plugins/navigator_pluginstest.cc

```diff
@@ -1,125 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#include "base/path_service.h"
```

Comment:
Are we planning to re-introduce `navigator.plugins`? I think updating the test makes more sense.

Reply:
@jumde I deleted existing tests because this PR deleted all related codes and went back to chromium's original state. If we want to control exposed plugin/mimetype list strictly, I think we need to re-implement instead of previous `BravePluginRegistryImpl` way. (See https://github.com/brave/brave-core/pull/2128#issuecomment-486500222). WDYT?